dotCMS/src/main/java/com/dotcms/rest/api/v1/template/TemplateForm.java

```diff
@@ -0,0 +1,299 @@
+    private final String  name;
+    private final String  title;
+    private final int     sortOrder;
+    private final boolean headerCheck;
+    private final boolean footerCheck;
+
+    private TemplateForm(final Builder builder) {
+
+        this.identifier = builder.identifier;
+        this.inode = builder.inode;
+        this.body = builder.body;
+        this.selectedimage = builder.selectedimage;
+        this.image = builder.image;
+        this.drawed = builder.drawed;
+        this.showOnMenu = builder.showOnMenu;
+        this.drawedBody = builder.drawedBody;
+        this.countAddContainer = builder.countAddContainer;
+        this.countContainers = builder.countContainers;
+        this.headCode = builder.headCode;
+        this.theme = builder.theme;
+        this.themeName = builder.themeName;
+        this.footer = builder.footer;
+        this.friendlyName = builder.friendlyName;
+        this.header = builder.header;
+        this.name = builder.name;
+        this.title = builder.title;
+        this.sortOrder = builder.sortOrder;
+        this.headerCheck = builder.headerCheck;
+        this.footerCheck = builder.footerCheck;
+    }
+
+    public String getIdentifier() {
+        return identifier;
+    }
+
+    public String getInode() {
+        return inode;
+    }
+
+    public String getBody() {
+        return body;
+    }
+
+    public String getSelectedimage() {
+        return selectedimage;
+    }
+
+    public String getImage() {
+        return image;
+    }
+
+    public boolean isDrawed() {
+        return drawed;
+    }
+
+    public boolean isShowOnMenu() {
+        return showOnMenu;
+    }
+
+    public String getDrawedBody() {
+        return drawedBody;
+    }
+
+    public int getCountAddContainer() {
+        return countAddContainer;
+    }
+
+    public int getCountContainers() {
+        return countContainers;
+    }
+
+    public String getHeadCode() {
+        return headCode;
+    }
+
+    public String getTheme() {
+        return theme;
+    }
+
+    public String getThemeName() {
+        return themeName;
+    }
+
+    public String getFooter() {
+        return footer;
+    }
+
+    public String getFriendlyName() {
+        return friendlyName;
+    }
+
+    public String getHeader() {
+        return header;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public String getTitle() {
+        return title;
+    }
+
+    public int getSortOrder() {
+        return sortOrder;
+    }
+
+    public boolean isHeaderCheck() {
+        return headerCheck;
+    }
+
+    public boolean isFooterCheck() {
+        return footerCheck;
+    }
+
+    public static final class Builder {
+
+        @JsonProperty
+        private  String identifier;
+        @JsonProperty
+        private  String inode;
+        @JsonProperty
+        private  String body;
+        @JsonProperty
+        private  String selectedimage;
+        @JsonProperty
+        private  String image;
```

Comment:
Codacy found an issue: [Field image has the same name as a method](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6042990)